encrypted-media-respec.html

```diff
@@ -591,7 +591,7 @@ <h2>Methods</h2>
                 Implementations MUST meet all related requirements and SHOULD follow related recommendations such that the risks on in an secure context would be similar.
               </p>
             </div>
-            <p>When this method is invoked, the user agent must run the following steps:</p>
+            <p>When this method is invoked, the user agent MUST run the following steps:</p>
```

Comment:
The same change needs to be applied to other occurrences of that sentence in the spec, see https://github.com/w3c/encrypted-media/pull/530#discussion_r1572157595